### PR TITLE
Add AES-CMAC algorithm support in Windows port

### PR DESCRIPTION
**Pull request by aggarw13**
Make changes equivalent to POSIX port for AES-CMAC functionality made in PR #97 in the Windows PAL file.

### CODE REVIEW

**Review by paulbartell — APPROVED** [review]

**Comment by aggarw13** [comment]
The CBMC proof failure is unrelated to the changes in this PR. The proof failure can be fixed separately.

File: source/portable/windows/core_pkcs11_pal.c [hunk 1]
```diff
@@ -141,6 +143,13 @@ void prvLabelToFilenameHandle( uint8_t * pcLabel,
             *pcFileName = pkcs11palFILE_CODE_SIGN_PUBLIC_KEY;
             *pHandle = eAwsCodeSigningKey;
         }
+        else if( 0 == strncmp( pkcs11configLABEL_CMAC_KEY,
```
Comment (aggarg) [inline source/portable/windows/core_pkcs11_pal.c:146]: Why not memcmp as in other conditions?

Comment (aggarw13) [inline source/portable/windows/core_pkcs11_pal.c:146]:
```suggestion
        else if( 0 == memcmp( pkcs11configLABEL_CMAC_KEY,
```

Comment (aggarw13) [reply source/portable/windows/core_pkcs11_pal.c]: I took these changes of AES CMAC support from the POSIX port of corePKCS11 PAL that uses `strncmp` (as can be seen [here](https://github.com/FreeRTOS/corePKCS11/blob/8a104f3e28a96f41813e67e5271ae756ea004e04/source/portable/posix/core_pkcs11_pal.c#L153-L158)). There is some duplication of functionality between the Windows and POSIX ports, something that can be hygiened into a common file to reduce maintenance cost.

**Review by aggarg — APPROVED** [review]

**Review by aggarg — APPROVED** [review]

**Pull request merged**